dc/council/code/titles/9/sections/9-1103.01.xml

```diff
@@ -332,10 +341,10 @@
         <num>(h)</num>
         <text>The exercise of any power of the discharge of any duty conferred or imposed upon the State Corporation Commission of Virginia by the Virginia Constitution.</text>
       </para>
+      <aftertext>
```

Comment:
The way I'm reviewing, I can't see the previous comment. But having these as aftertext's  seems semantically incorrect. These are more like `<heading>` for the `<num>` that follow.